### ISSUE TITLE
docs: Should mention foo-*.example.com as possible WILDCARD_HOST strategy

### ISSUE THREAD

**Issue by paulproteus** [sandstorm-io/sandstorm#1567]
https://github.com/sandstorm-io/sandstorm/pull/1541 doesn't help people who want to use Sandstorm on an existing HTTPS wildcard.

**Comment by zeigerpuppy** [comment]
good idea but not valid with some DNS software (djbdns for instance does not regard this a s a wildcard).
I managed to use

*ss.example.com with good success
